### PR TITLE
Endorsements: add Open Collective

### PR DESCRIPTION
**Pull request by vladh**

### CODE REVIEW

File: src/components/EndorsementList.astro [hunk 1]
```diff
@@ -21,6 +21,15 @@ const { full } = Astro.props;
...
   <p>Every modern business—whether they know it or not—relies on Open Source software, yet few give back. The PHP Foundation celebrates the #OpenSourcePledge, whose members have committed to correcting this imbalance. Together, we can build a stronger, more resilient OSS ecosystem. 🌱</p>
   </Endorsement>
+  <Endorsement
+    name="Open Collective"
+    handle="@opencollect"
+    imageUrl="/images/endorsers/opencollective.svg"
+    sourceUrl="https://blog.opencollective.com/why-the-open-source-pledge-is-both-relevant-and-timely/"
+    hidden={!full}
+  >
+  <p>[The Open Source Pledge's] simple message is incredibly important. Companies: pay your share.</p>
```
[the remark this answers is not in the document]
Comment (vladh) [reply src/components/EndorsementList.astro:31]: Haha, let's go for it.

[the remark this answers is not in the document]
Comment (chadwhitacre) [reply src/components/EndorsementList.astro]: Oh! Also we need to credit Open _Source_ Collective, not Open Collective. I have this in email from last week:

> Please change any 'Open Collective' reference to 'Open Source Collective'. We are two separate business entities although we work closely together. Open Source Collective (OSC) is the nonprofit fiscal host of open source projects and Open Collective (OC) is the software we use to facilitate fundraising and expense processing.

[the remark this answers is not in the document]
Comment (vladh) [reply src/components/EndorsementList.astro]: Hmm…that's also what I thought, but this post is specifically on the _Open Collective_ blog… 🤔

[the remark this answers is not in the document]
Comment (vladh) [reply src/components/EndorsementList.astro]: Oh nevermind, it's a bit confusing — the author is listed as “Open Collective”, but the post refers to the Open _Source_ Collective. I'll change it.

[the remark this answers is not in the document]
Comment (chadwhitacre) [reply src/components/EndorsementList.astro]: I agree it's confusing! 🙈

**Pull request merged**